.github/workflows/publish.yaml

```diff
@@ -120,15 +104,8 @@ jobs:
           submodules: ${{ inputs.checkout_submodules }}
 
       - uses: subosito/flutter-action@44ac965b96f18d999802d4b807e3256d5a3f9fa1
-        if: ${{ inputs.use-flutter }}
         with:
           channel: ${{ inputs.sdk }}
-          
-
-      - uses: dart-lang/setup-dart@0a8a0fc875eb934c15d08629302413c671d3f672
-        if: ${{ !inputs.use-flutter }}
-        with:
-          sdk: ${{ inputs.sdk }}
 
       - name: Install firehose
         run: dart pub global activate firehose
```

Comment:
@devoncarew @mosuem Have we made breaking changes to `firehose` CLI before? How do we handle cases where a repo could be using an older `publish.yaml` file than what `dart pub global activate firehose` installs?

Reply:
We recommend using `@main`, so as long as we publish shortly after landing this on the main branch, I think it should work out for most use cases.

Is it OK if we break configuration that are pinning the publish workflow?

Reply:
I don't believe we've made breaking CLI changes before.

Repos generally always reference the `.github/workflows/publish.yaml` script at head (i.e., `uses: dart-lang/ecosystem/.github/workflows/publish.yaml@main`). That then globally activates firehose - that latest published version. We update the various publish.yaml clients to remove any use of `use-flutter`, but I don't think we'll have any other work to do.

Reply:
I think breaks are ok; we should have some good text for the changelog, and likely rev. to a new 'major' version to help call out the change.